zerver/openapi/zulip.yaml

```diff
@@ -1557,6 +1557,10 @@ paths:
                                   type: boolean
                                   description: |
                                     Whether the user has marked themself "away" with this status.
+
+                                    **Changes**: As of Zulip 6.0 (feature level 147), `away` is a
+                                    deprecated way to access a user's `presence_enabled` setting
+                                    for clients supporting older servers, with `away!=presence_enabled`.
```

Comment:
I think `away = !presence_enabled` would be a cleaner way to spell that conditional.

I would write it more like this -- we usually try to have the sentence start with `Deprecated` to make that detail stand out for someone skimming:

**Changes**: Deprecated in Zulip 6.0 (feature level 147); starting with that version,
`away` is a legacy way to access the user's `presence_enabled` setting, with 
`away = !presence_enabled`. To be removed in a future release.